README.md

```diff
@@ -0,0 +1,19 @@
+# PokemonTCGApp
+
+Hello, my name is Marcos Cifuentes, and I'm currently a trainee at Globant in Mobile Studio.
+
+As part of my training, my task was to create an app based on the [POKEMON TCG API](https://docs.pokemontcg.io/#documentationgetting_started), that returns information abut Pokemon cards.
```

Comment:
change this for something more generic.. like **This project is about to create an app based on the [POKEMON TCG API]**

Reply:
done, thanks